modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -2875,6 +2891,60 @@ else if (objectField.isSystem()) {
 		}
 	}
 
+	private void _updateResourcePermissions(
+			ObjectDefinition objectDefinition, ServiceContext serviceContext)
+		throws PortalException {
+
+		ModelPermissions modelPermissions =
+			serviceContext.getModelPermissions();
+
+		if (modelPermissions == null) {
+			return;
+		}
+
+		Company company = _companyLocalService.getCompany(
+			objectDefinition.getCompanyId());
+
+		_permissionService.checkPermission(
```

Comment:
@vngoliveira it's necessary to use it the groupId or we could use it null like in the `DefinitionServiceImpl`?